components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/js/operator-finder/operator-finder.js

```diff
@@ -304,6 +306,43 @@ define(['jquery', 'lodash', 'log','remarkable', 'handlebar', 'designViewUtils',
             }
         };
 
+        /**
+         * functions to get the not installed extension array details.
+         */
+        var getNotInstalledExtensionDetails = function () {
```

Comment:
change to getUninstalledExtensionDetails

Reply:
88cbc98 this commit will fix this change request

Reply:
@dnwick Shall we say `getNonInstalledExtensionDetails` ? 'Uninstall' might be misinterpreted as "Installed, and removed later"

Reply:
since @dnwick  asked me to change that as `getUninstalledExtensionDetails`,I have changed as that.If he ok with this i will be ok